homeassistant/helpers/script.py

```diff
@@ -634,6 +634,7 @@ def log_cb(level, msg, **kwargs):
             self._log(msg, level=level, **kwargs)
 
         to_context = None
+        done = asyncio.Event()
```

Comment:
We need to initialize the event before the call to `async_initialize_triggers` below in case the trigger fires immediately.

Reply:
Since it's referenced in async_done, maybe even more clear to define prior to that?

Reply:
Fixed, agreed it makes it clearer.